### ISSUE TITLE
checkIfUnsafeContent causes too many false positives

### ISSUE THREAD

**Issue by xanoni** [standardnotes/bold-editor#91]
I noticed that after pasting text from websites into Bold Editor, I often get a message like the below.

<img width="458" alt="image" src="https://user-images.githubusercontent.com/77220130/133741752-73205b3c-67e6-4e99-b2c0-307f473332e4.png">

I looked at the text content in my notes and this seems to be related to harmless HTML attributes that get dropped (e.g., `target=x` in `<a>` tags), which will make the comparison below fail ... `renderedDom` will be != `sanitizedDom`. I didn't have more time to look at it, but I assume that there's a very narrow whitelist of allowed attributes that doesn't cover all the things that we see on the legit web.

https://github.com/standardnotes/bold-editor/blob/01f03e29bce7b8a8f72e23b17d9e57587b2481c6/app/components/Editor.js#L277-L318

**Comment by mobitar** [comment]
Once you press Continue once on a note, it should remember that preference and not bother you about it again. Is that the case? I hear you on the original issue but just want to make sure this is the case as well.

**Comment by xanoni** [comment]
Yes that is the case. But it's not good if the app conditions users to just ignore security warnings :)

**Comment by JaspalSuri** [comment]
Thank you for the additional feedback on how the UX can be improved for this feature. I'll forward it over to the dev team to review. 🙂

(Thank you for your support as well!)

**Comment by xanoni** [comment]
Keep  rocking.
